### PR TITLE
FIX: When user uses custom electrum server, make him aware that his a…

### PR DESCRIPTION
**Pull request by marcosrdz**
…ddresses/txids sre still leaked to a Push Notificatinos serer #4806
signal-2024-10-04-075411_002

## Summary by CodeRabbit

- **New Features**
	- Introduced a new Notification Settings component for managing notification preferences, including enabling/disabling notifications and inputting a user-defined URI.
	- Added detailed explanations for push notifications to enhance user understanding.
	- Enhanced the action sheet message for receiving notifications to provide more context.

- **Bug Fixes**
	- Corrected the import casing for the Notification Settings component to ensure proper functionality.

- **Improvements**
	- Reorganized localization keys for better clarity and structure.

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: loc/en.json [hunk 1]
```diff
@@ -302,8 +302,8 @@
     "privacy_clipboard_explanation": "Provide shortcuts if an address or invoice is found in your clipboard.",
     "privacy_do_not_track": "Disable Analytics",
     "privacy_do_not_track_explanation": "Performance and reliability information will not be submitted for analysis.",
-    "push_notifications": "Push Notifications",
     "rate": "Rate",
+    "push_notifications_explanation": "By enabling notifications, your device token will be sent to our server, along with wallet addresses and transaction IDs for all wallets and transactions made after enabling notifications. The device token is used to send notifications, and the wallet information allows us to notify you about incoming Bitcoin or transaction confirmations.\n\nOnly information from after you enable notifications is transmitted—nothing from before is collected.\n\nDisabling notifications will remove all of this information from the server. Additionally, deleting a wallet from the app will also remove its associated information from the server.",
```
Comment (Overtorment) [inline loc/en.json:306]: to our server  ->  to the server

**Review by Overtorment — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**